src/test/string-utils.test.js

```diff
@@ -0,0 +1,45 @@
+const assert = require('assert');
+
+const StringUtils = require('../main/lib/string-utils');
+
+describe('StringUtils', () => {
+  describe('splitBy', () => {
```

Comment:
What is that? It is not method name, is it? Shouldn't that be `split` which corresponds to method name and then test cases would just say `it('returns empty array for undefined input', () => {`

Reply:
Yeah it was originally splitBy.. fixing!